graph/telemetry/istio/appender/response_time.go

```diff
@@ -27,6 +27,9 @@ var (
 // is represented as a percentile value. The default is 95th percentile, which means that
 // 95% of requests executed in no more than the resulting milliseconds. ResponeTime values are
 // reported in milliseconds.
+// Response Times are reported using destination proxy telemetry, when available, which should remove
+// network latency fluctuations.
+// TODO: Should we report both source and destination when possible?
```

Comment:
As we could fetch both response time from source and also response time from destination, rather than reporting both, perhaps we could report something similar to "Jitter"?

See: https://en.wikipedia.org/wiki/Jitter#Packet_jitter_in_computer_networks

Possibly related to: #2887

Reply:
@israel-hdez It's something to look into and definitely relevant to #2887.  It's not directly relevant to this PR.  I think the first thing to verify is the assumption that source proxies do in fact include latency and dest proxies do not.  I have been assuming yes, but it's something to make sure of before we do any work.

Reply:
I fully about verifying that source proxies include latency.